doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,590 @@
+    * :ref:`gs_programming` - restructured the content and added information about building on the command line
+    * :ref:`gs_modifying` - updated the content and added information about configuring an application
+
+  * User guides:
+
+    * :ref:`ug_nrf5340` - added
+    * :ref:`ug_thingy91` - added
+    * :ref:`ug_ble_controller` - added
+    * :ref:`ug_multi_image` - updated with content that was removed from the Zephyr fork
+
+  * nrfxlib:
+
+    * :ref:`nrfxlib:bsdlib` - extended and restructured the content
+    * :ref:`nrfxlib:mpsl` - added
+    * :ref:`nrfxlib:ble_controller_readme` - updated to match current version of the nRF Bluetooth LE Controller
+
+
+
+Known issues
+************
+
+nRF9160
+=======
+
+* The :cpp:func:`nrf_send` function in the :ref:`nrfxlib:bsdlib` might be blocking for several minutes, even if the socket is configured for non-blocking operation.
+  The behavior depends on the cellular network connection.
+* The :ref:`gps_with_supl_support_sample` sample stops working if :ref:`supl_client` support is enabled, but the SUPL host name cannot be resolved.
+  As a workaround, insert a delay (``k_sleep()``) of a few seconds after the ``printf`` on line 294 in :file:`main.c`.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+* nRF91 fails to receive large packets (over 4000 bytes).
+* nrf_connect fails if called immediately after initialization of the device.
+  A delay of 1000 ms is required for this to work as intended.
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* Bluetooth LE cannot be used in a non-secure application, for example, an application built for the nrf5340_dk_nrf5340_cpuappns board.
+  Use the nrf5340_dk_nrf5340_cpuapp board instead.
+* The :ref:`peripheral_hids_keyboard` sample cannot be used with the :ref:`nrfxlib:ble_controller` because the NFC subsystem does not work with the controller library.
+  The library uses the MPSL Clock driver, which does not provide an API for asynchronous clock operation.
+  NFC requires this API to work correctly.
+* When the :ref:`peripheral_hids_mouse` sample is used with the Zephyr Bluetooth LE Controller, directed advertising does not time out and the regular advertising cannot be started.
+* The :ref:`bluetooth_central_hids` sample cannot connect to a peripheral that uses directed advertising.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed and then disconnections/re-connections happen).
```

Comment:
I think since the last NCS release, most of the stability issues in BLE samples have been fixed. Most of them were caused by incorrect default configuration. @shantha-14 can we remove this point?

Reply:
Removed.